src/gui_common/tooltip/microbe_editor/SelectionMenuToolTip.cs

```diff
@@ -306,6 +323,15 @@ private void UpdateMpCost()
         mpLabel.Text = ((int)(mpCost * editorCostFactor)).ToString(CultureInfo.CurrentCulture);
     }
 
+    private void UpdateRequiresNucleus()
+    {
+        if (requiresNucleusLabel == null)
+            return;
+
+        requiresNucleusLabel.Text = TranslationServer.Translate("REQUIRES_NUCLEUS");
```

Comment:
One small comment from me:
The label text seems to be always the same, so why isn't it set in the scene file?